### PR TITLE
Edit task

### PR DESCRIPTION
**Pull request by SvenjaKlenke**

### CODE REVIEW

File: backend/src/main/java/de/neuefische/backend/controller/ControllerTasks.java [hunk 1]
```diff
@@ -28,4 +28,9 @@ public TaskModel addNewTask(@RequestBody TaskModel taskModel) {
     public void deleteById(@PathVariable String id) {
         serviceTasks.deleteById(id);
     }
+
+    @PutMapping({"/{id}"})
+    public TaskModel editTask(@PathVariable String id, @RequestBody TaskModel taskModel) {
+        return serviceTasks.editTask(taskModel);
```
Comment (KitNaIn) [inline backend/src/main/java/de/neuefische/backend/controller/ControllerTasks.java:34]: is there a reason you're not using a DTO here?

File: backend/src/test/java/de/neuefische/backend/controller/ControllerTasksTest.java [hunk 2]
```diff
@@ -91,4 +91,60 @@ void deleteToDo_thenReturn200OK_andReturnEmptyList() throws Exception {
...
                 .andExpect(content().json("[]"));
     }
-}
\ No newline at end of file
+
+    @Test
+    @DirtiesContext
+    void editTask_thenReturn200OK_andReturnTheEditTask() throws Exception {
+        MvcResult result = mockMvc.perform(MockMvcRequestBuilders.post("/tasks")
+                        .contentType(MediaType.APPLICATION_JSON)
```
Comment (KitNaIn) [inline backend/src/test/java/de/neuefische/backend/controller/ControllerTasksTest.java:99]: is there no verify or .andExpect needed here?

**Review by KitNaIn — COMMENTED** [review]
looks well structured so far, well done! Just check the tests again :)

File: backend/src/test/java/de/neuefische/backend/controller/ControllerTasksTest.java (hunk 2 above)
Comment (SvenjaKlenke) [reply backend/src/test/java/de/neuefische/backend/controller/ControllerTasksTest.java:99]: andExpect is in line 112 and i test post in another test

File: backend/src/main/java/de/neuefische/backend/controller/ControllerTasks.java (hunk 1 above)
Comment (SvenjaKlenke) [reply backend/src/main/java/de/neuefische/backend/controller/ControllerTasks.java:34]: i dont think i needed it here.

**Review by KitNaIn — APPROVED** [review]

**Pull request merged**